charts/newrelic-infrastructure/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart to deploy the New Relic Infrastructure Agent as a DaemonSet
 name: newrelic-infrastructure
-version: 2.7.3
-appVersion: 2.8.3
+version: 2.7.4
+appVersion: 2.8.4
```

this version should be a valid nri-kubernetes [released](https://github.com/newrelic/nri-kubernetes/releases) version 
```suggestion
appVersion: 2.8.3
```